styleguide/style.md

```diff
@@ -120,10 +120,25 @@ Lack of spacing makes it a little harder to read in source:
 Text after... DO NOT DO THIS.
 ```
 
+### Capitalization
+
 Use sentence-style capitalization. Always capitalize:
 - The first word of a heading.
 - The word following a colon in a title or heading (for example, "How to: Sort an array").
 
+### Verb form
```

Comment:
I think you don't need to create a new section.
You can explain it in a new paragraph.

Reply:
Thanks for the good reference, 
but aren't "starting sentence with verb form" and "capitalization" different issues?

I hoped to add "use verb for the heading",
but your reference seems to enhancing capitalization section.

Reply:
The reference is about "title and heading capitalization". I want you to update this paragraph as above.
Authors can use not only verb phrase but also noun phrase.
It's okay. I will update this section later.

Reply:
> The reference is about "title and heading capitalization". I want you to update this paragraph as above.

Does this comment mean to create a new PR to update the "### Capitalization" section like below?

### Capitalization

Use sentence-style capitalization. Always capitalize:
- The first word of a heading.
- The word following a colon in a title or heading (for example, "How to: Sort an array").

```markdown
Capitalize first word and proper nouns
```

```markdown
Capitalize First Word and Proper Nouns...DO NOT DO THIS
```

If so, I'll create a new PR.